src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -671,6 +848,16 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
         if (!this._options.justTypes) {
             this.emitLine("using nlohmann::json;");
             this.ensureBlankLine();
+
+            this.emitLine("template <typename T>");
+            this.emitLine("std::string stringify(const T &t)");
```

Comment:
`emitBlock` or `emitMultiline`.  The latter automatically converts the indentation in the multiline string (has to be 4 spaces) to whatever is set in the target language.